src/components/ActivityButton.tsx

```diff
@@ -0,0 +1,18 @@
+import { Button } from '@mui/material';
+
+type ActivityButtonProps = {
+    color?: "inherit" | "primary" | "secondary";
```

Comment:
I dont see the colors added anywhere while using ActivityButton. So maybe at this moment we dont need them?

Reply:
Used the secondary color on the button. So didn't remove it